policy/modules/system/systemd.if

```diff
@@ -30,8 +30,19 @@ template(`systemd_role_template',`
 		attribute systemd_user_session_type, systemd_log_parse_env_type;
 		type systemd_user_runtime_t, systemd_user_runtime_notify_t;
 		type systemd_run_exec_t, systemd_analyze_exec_t;
+		type systemd_user_home_t, systemd_user_home_unit_t;
+		type systemd_user_runtime_unit_t;
+		class dbus { acquire_svc send_msg };
 	')
 
+	## <desc>
+	##	<p>
+	##	Grant the $1 user access to manage systemd user
+	##  units in the transient scope.
+	##	</p>
+	## </desc>
+	gen_tunable(`systemd_$1_manage_transient_units', false)
+
```

Comment:
not sure if this is worth a boolean might even break things?

Reply:
This was an attempt to see if I could address the potential privilege escalation when using `dbus-broker` as discussed in the issue, but you're right. I think it would be best to remove this boolean and address it later since this does indeed break usage of `systemd-run` for example even when using `dbus-daemon`.